signlink.css

```
@@ -0,0 +1,138 @@
+* {
+    box-sizing: border-box;
+  }
+  
+  html {
+    word-break: keep-all;
+  }
+  
+  body {
+    margin:50px auto;
+  }
+
+
+  header{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    flex-shrink: 0;
+  }
+
+  label {
+    color: #2c2c2c;
+    display: block;
+    color: var(--Secondary-800, #1F2937);
```

P2:
색상 변수 언언을 안하셨다면 아래처럼 사용해주시는 것이 가독성에 더 좋을 것 같습니다.

```suggestion
    color: #1F2937;
```